plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -295,17 +302,23 @@ def sshKeyExists(self, user, sshkey_name):
 
     '''
     @param user User object
-    @param sshkey Dict containing sshkey infos {"name": "", "file": ""}
+    @param sshkey Dict containing sshkey infos {"name": "", "file": "", "expires_at": ""}
     '''
     def addSshKeyToUser(self, user, sshkey):
         if not self.sshKeyExists(user, sshkey['name']):
             if self._module.check_mode:
                 return True
 
             try:
-                user.keys.create({
-                    'title': sshkey['name'],
-                    'key': sshkey['file']})
+                if sshkey['expires_at'] is None:
+                    user.keys.create({
+                        'title': sshkey['name'],
+                        'key': sshkey['file']})
+                else:
+                    user.keys.create({
+                        'title': sshkey['name'],
+                        'key': sshkey['file'],
+                        'expires_at': sshkey['expires_at']})
```

Comment:
How about changing this to:
```suggestion
                parameter = {
                    'title': sshkey['name'],
                    'key': sshkey['file'],
                }
                if sshkey['expires_at'] is not None:
                    parameter['expires_at'] = sshkey['expires_at']
                user.keys.create(parameter)
```